core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -216,7 +216,8 @@ class LogManager(logDirs: Array[File],
       maxProducerIdExpirationMs = maxPidExpirationMs,
       scheduler = scheduler,
       time = time,
-      brokerTopicStats = brokerTopicStats)
+      brokerTopicStats = brokerTopicStats,
+      logDirFailureChannel = logDirFailureChannel)
```

Comment:
@ijuma Sure. I have updated the patch to remove all default arguments in `Log.apply`. Can you take a look? Thanks!